lib/ts/unit-tests/test_string_view.cc

```diff
@@ -256,8 +256,8 @@ TEST_CASE("Access & iterators", "[string_view] [access]")
 
     REQUIRE(*sv.begin() == 'a');
     REQUIRE(*sv.cbegin() == 'a');
-    REQUIRE(*(--sv.end()) == 'e');
```

Comment:
Do we even want to unit test string_view with c++ 17? Given that at this level we are using the standard one?

Reply:
I think it would be nice to run the string_view unit test to make sure they are compatible.